swagger/v1/move.json

```diff
@@ -63,6 +63,16 @@
               { "type": "null" }
             ],
             "description": "Additional information about the move that the supplier should be made aware of"
+          },
+          "cancellation_reason": {
+            "type": "string",
+            "enum": ["error", "null", "other", "supplier_cannot_move"],
```

Comment:
Do we need to include `null` as an option in the enum? Or can we state that the attribute can also be null?

Reply:
To be honest I'm not sure, I will change it to be consistent with the other property